code/__DEFINES/achievements.dm

```diff
@@ -4,33 +4,35 @@
 #define ACHIEVEMENT_SCORE "score"
 
 //Misc Medal hub IDs
-#define MEDAL_METEOR 			"Your Life Before Your Eyes"
-#define MEDAL_PULSE 			"Jackpot"
-#define MEDAL_TIMEWASTE 		"Overextended The Joke"
-#define MEDAL_RODSUPLEX 		"Feat of Strength"
-#define MEDAL_CLOWNCARKING 		"Round and Full"
-#define MEDAL_THANKSALOT 		"The Best Driver"
-#define MEDAL_HELBITALJANKEN	"Hel-bent on Winning"
-#define MEDAL_MATERIALCRAFT 	"Getting an Upgrade"
-#define MEDAL_DISKPLEASE	 	"Disk, Please!"
-#define MEDAL_GAMER			 	"I'm Not Important"
-#define MEDAL_VENDORSQUISH		"Teenage Anarchist"
-#define MEDAL_SWIRLIE			"Bowl-d"
-#define MEDAL_SELFOUCH			"Hands???"
-#define MEDAL_SANDMAN			"Mister Sandman"
-#define MEDAL_CLEANBOSS			"Cleanboss"
-#define MEDAL_RULE8				"Rule 8"
-#define MEDAL_LONGSHIFT			"longshift"
-#define MEDAL_SNAIL				"KKKiiilll mmmeee"
-#define MEDAL_LOOKOUTSIR		"Look Out, Sir!"
-#define MEDAL_GOTTEM			"GOTTEM"
-#define MEDAL_ASCENSION			"Ascension"
-#define MEDAL_FRENCHING			"FrenchingTheBubble"
-#define MEDAL_ASH_ASCENSION		"Ash"
-#define MEDAL_FLESH_ASCENSION	"Flesh"
-#define MEDAL_RUST_ASCENSION	"Rust"
-#define MEDAL_VOID_ASCENSION	"Void"
-#define MEDAL_TOOLBOX_SOUL		"Toolsoul"
+#define MEDAL_METEOR "Your Life Before Your Eyes"
+#define MEDAL_PULSE "Jackpot"
+#define MEDAL_TIMEWASTE "Overextended The Joke"
+#define MEDAL_RODSUPLEX "Feat of Strength"
+#define MEDAL_CLOWNCARKING "Round and Full"
+#define MEDAL_THANKSALOT "The Best Driver"
+#define MEDAL_HELBITALJANKEN "Hel-bent on Winning"
+#define MEDAL_MATERIALCRAFT "Getting an Upgrade"
+#define MEDAL_DISKPLEASE "Disk, Please!"
+#define MEDAL_GAMER "I'm Not Important"
+#define MEDAL_VENDORSQUISH "Teenage Anarchist"
+#define MEDAL_SWIRLIE "Bowl-d"
+#define MEDAL_SELFOUCH "Hands???"
+#define MEDAL_SANDMAN "Mister Sandman"
+#define MEDAL_CLEANBOSS "Cleanboss"
+#define MEDAL_RULE8 "Rule 8"
+#define MEDAL_LONGSHIFT "longshift"
+#define MEDAL_SNAIL "KKKiiilll mmmeee"
+#define MEDAL_LOOKOUTSIR "Look Out, Sir!"
+#define MEDAL_GOTTEM "GOTTEM"
+#define MEDAL_ASCENSION "Ascension"
+#define MEDAL_FRENCHING "FrenchingTheBubble"
+#define MEDAL_ASH_ASCENSION "Ash"
+#define MEDAL_FLESH_ASCENSION "Flesh"
+#define MEDAL_RUST_ASCENSION "Rust"
+#define MEDAL_VOID_ASCENSION "Void"
+#define MEDAL_TOOLBOX_SOUL "Toolsoul"
+//#define MEDAL_CHEM_TUT "Beginner Chemist"
+#define MEDAL_HOT_DAMN "Hot Damn!"
```

Comment:
Would like these to get their unified indentation back (Where they're all on the same column)